src/app/transaction-spending-limit/transaction-spending-limit-section/transaction-spending-limit-section.component.html

```diff
@@ -134,9 +166,12 @@
   <div
     *ngIf="showDetails"
     class="d-flex justify-content-center mr-15px ml-15px"
-    style="background: lightgray; border: 1px solid darkgray; border-radius: 2px"
-    (click)="showDetails = !showAll; showAll = !showAll || getSectionMapLength() <= defaultNumShown;"
+    style=""
+    (click)="
+      showDetails = !showAll;
+      showAll = !showAll || getSectionMapLength() <= defaultNumShown
+    "
   >
-    {{ showAll ? "Hide" : "Show all" }} {{ getSectionMapLength() }} {{ sectionItemType() }}s
+    <!-- {{ showAll ? "Hide" : "Show all" }} {{ getSectionMapLength() }} {{ sectionItemType() }}s -->
```

Comment:
What was this showing previously?

Reply:
It's an additional button that handles the same logic as the arrow component. Seemed redundant to have so makes sense to remove it IMO